src/sentry/relay/config/__init__.py

```diff
@@ -155,20 +156,13 @@ def get_project_config(project, full_config=True, project_keys=None):
 
 
 def get_dynamic_sampling_config(project) -> Optional[Mapping[str, Any]]:
-    allow_server_side_sampling = features.has(
-        "organizations:server-side-sampling",
-        project.organization,
-    )
-    allow_dynamic_sampling = features.has(
-        "organizations:dynamic-sampling",
-        project.organization,
-    )
+    feature_multiplexer = DynamicSamplingFeatureMultiplexer(project, None)
```

Comment:
Not part of this PR, but we could probably use `features.batch_has` in the multiplexer's `__init__`.

Reply:
yeah good idea.